packages/styles/scss/themes/expressive/components/_button.scss

```diff
@@ -15,7 +15,43 @@
 /// @access private
 /// @group button-expressive
 @mixin button-expressive {
+  .#{$prefix}--btn {
+    padding: calc(0.875rem - 4px) 61px calc(0.875rem - 4px) 13px;
+
+    &.#{$prefix}--btn--ghost {
+      padding: calc(0.875rem - 4px) 13px;
+    }
+
+    .#{$prefix}--btn__icon {
+      width: auto;
+      height: auto;
+    }
+  }
+
+  .#{$prefix}--btn--field,
   .#{$prefix}--btn--sm {
-    min-height: rem(40px);
+    padding: 6px 61px 6px 13px;
+
+    &.#{$prefix}--btn--ghost {
+      padding: 6px 13px;
```

Comment:
@annawen1 @oliviaflory this is some pretty specific padding, will we run into issues when we try to contribute this back to carbon?

Reply:
hmm I could convert it to rems? 

Although I do see specific padding on their end too:
```
/// @type Value
/// @access public
/// @group button
$button-padding-field: calc(0.675rem - 3px) 60px calc(0.675rem - 3px) 12px !default;
```

Reply:
Looks like they set a default $button-padding-field variable that's used on their end. Would we be able to set variables on our end (to match whatever variables they are using) and set it on our end?

Reply:
yeah I could do that

Reply:
yeah i could do that